ggml/src/gguf.cpp

```diff
@@ -934,6 +934,8 @@ static void gguf_check_reserved_keys(const std::string & key, const T val) {
         } else {
             GGML_ABORT(GGUF_KEY_GENERAL_ALIGNMENT " must be type u32");
```

Comment:
I think it makes more sense to put the `GGML_UNUSED` above this line since this is where the `if constexpr` happens.